.github/workflows/test.yml

```diff
@@ -24,6 +24,22 @@ jobs:
           node-version: ${{ matrix.node }}
       - run: npm ci
       - run: npm test
+  typescript_matrix:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        typescript:
+          - 4.1
+          - 4.2
+    name: TypeScript ${{matrix.typescript}}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 14
+      - run: npm ci
+      - run: npm i typescript@${{matrix.typescript}}
+      - run: npm test
```

Comment:
Are there other places in Octokit ecosystem where this job would have sense? Could it be interesting to create the GitHub Action and extend it in all the workflows we consider it necessary?

Reply:
It would probably make sense to use this in `@octokit/webhooks-definitions` as well. The other projects, it could be useful, but the TS version range needs to be changed per project (usually `>=4.0, < 5`)

Reply:
> It would probably make sense to use this in `@octokit/webhooks-definitions` as well. The other projects, it could be useful, but the TS version range needs to be changed per project (usually `>=4.0, < 5`)

Sounds like a mission for [Octoherd](https://github.com/octoherd) team 💪🏽

Reply:
Sure. Let's maybe wait for @gr2m to see what he thinks before we move forward